src/solvers/flattening/boolbv_add_sub.cpp

```diff
@@ -27,71 +27,52 @@ bvt boolbvt::convert_add_sub(const exprt &expr)
      type.id()!=ID_vector)
     return conversion_failed(expr);
 
-  std::size_t width=boolbv_width(type);
-
+  const std::size_t width = boolbv_width(type);
   if(width==0)
     return conversion_failed(expr);
-
   const exprt::operandst &operands=expr.operands();
 
-  if(operands.empty())
-    throw "operator "+expr.id_string()+" takes at least one operand";
-
+  INVARIANT(
+    !operands.empty(),
+    "operator " + expr.id_string() + " takes at least one operand");
   const exprt &op0=expr.op0();
 
-  if(op0.type()!=type)
-  {
-    std::cerr << expr.pretty() << '\n';
-    throw "add/sub with mixed types";
-  }
-
+  INVARIANT(op0.type() == type, "add/sub should not have with mixed types");
   bvt bv=convert_bv(op0);
-
-  if(bv.size()!=width)
-    throw "convert_add_sub: unexpected operand 0 width";
-
-  bool subtract=(expr.id()==ID_minus ||
-                 expr.id()=="no-overflow-minus");
-
-  bool no_overflow=(expr.id()=="no-overflow-plus" ||
-                    expr.id()=="no-overflow-minus");
-
-  typet arithmetic_type=
-    (type.id()==ID_vector || type.id()==ID_complex)?
-      ns.follow(type.subtype()):type;
-
-  bv_utilst::representationt rep=
-    (arithmetic_type.id()==ID_signedbv ||
-     arithmetic_type.id()==ID_fixedbv)?bv_utilst::representationt::SIGNED:
-                                       bv_utilst::representationt::UNSIGNED;
+  INVARIANT(bv.size() == width, "convert_add_sub: unexpected operand 0 width");
+  const bool subtract =
+    (expr.id() == ID_minus || expr.id() == "no-overflow-minus");
+  const bool no_overflow =
+    (expr.id() == "no-overflow-plus" || expr.id() == "no-overflow-minus");
+
+  const typet arithmetic_type =
+    (type.id() == ID_vector || type.id() == ID_complex)
+      ? ns.follow(type.subtype())
+      : type;
+
+  const bv_utilst::representationt rep =
+    (arithmetic_type.id() == ID_signedbv || arithmetic_type.id() == ID_fixedbv)
+      ? bv_utilst::representationt::SIGNED
+      : bv_utilst::representationt::UNSIGNED;
```

Comment:
Is this all just reformatting?

Reply:
I was mainly adding `const` but clang-format then reformatted the whole instructions.